service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsController.java

```diff
@@ -66,7 +72,7 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
         return respond(caseDetails);
     }
 
-    @PostMapping("/get-fee/mid-event")
+    @PostMapping("/populate-data/mid-event")
```

Comment:
I would make sure to include the old mapping as well, if someone is trying to action this event when the code merges but before the ccd def is uploaded then they will get an event not found. You can do this by passing an array of your endpoints, i.e. `    @PostMapping({"...", "..."})`